Missing include of TopoDS_Edge.hxx in ShapeUpgrade_UnifySameDomain.hxx

References:
[1] https://forum.freecadweb.org/viewtopic.php?f=4&t=58090
[2] https://tracker.dev.opencascade.org/view.php?id=32328
[3] https://forum.freecadweb.org/viewtopic.php?f=10&t=20264

Author: looooo (GitHub)
Source: https://github.com/conda-forge/occt-feedstock/blob/4febc64/recipe/ShapeUpgrade.patch

diff --git a/src/ShapeUpgrade/ShapeUpgrade_UnifySameDomain.hxx b/src/ShapeUpgrade/ShapeUpgrade_UnifySameDomain.hxx
index b1558d11..0e2a83e5 100644
--- a/src/ShapeUpgrade/ShapeUpgrade_UnifySameDomain.hxx
+++ b/src/ShapeUpgrade/ShapeUpgrade_UnifySameDomain.hxx
@@ -21,7 +21,7 @@
 #include <Standard.hxx>
 #include <Standard_Type.hxx>
 
-#include <TopoDS_Shape.hxx>
+#include <TopoDS_Edge.hxx>
 #include <Standard_Boolean.hxx>
 #include <Standard_Transient.hxx>
 #include <TopTools_DataMapOfShapeShape.hxx>
